contracts/protocol/clients/FermionFractionsMint.sol

```diff
@@ -257,13 +257,11 @@ contract FermionFractionsMint is FermionFNFTBase, FermionErrors, FundsManager, I
                 emit FractionsMigrated(address(this), fractionBalance);
             }
             fractionStorage.migrated[address(this)] = true;
-        } else {
-            if (_owners.length == 0) {
-                revert InvalidLength();
-            }
-            cloneAddress = fractionStorage.epochToClone[0];
+        } else if (_owners.length == 0) {
+            revert InvalidLength();
         }
-
+        
+        cloneAddress = fractionStorage.epochToClone[0];
```

Comment:
I don't think that's even better - now, when `fractionStorage.epochToClone.length == 0` we are reading `fractionStorage.epochToClone[0]` twice and assigning it to `cloneAddress`.

If we could have `cloneAddress = fractionStorage.epochToClone[0];` only once, that'd be an improvement, but I overlooked that `cloneAddress` is already used in line 256 and therefore I think we cannot really improve what we already had. I think we should dismiss #495